CHANGELOG.md

```diff
@@ -384,6 +384,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Fixed error reporting in DDP process reconciliation when processes are launched by an external agent ([#9389](https://github.com/PyTorchLightning/pytorch-lightning/pull/9389))
 
 
+- Fixed missing deepspeed distributed call ([#9540](https://github.com/PyTorchLightning/pytorch-lightning/pull/9540))
```

Comment:
was #9540 included in the release? I can't see that it did.

Reply:
aha! great point :) seems like this will only affect 1.5, will change the milestone!